components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -155,16 +155,36 @@ esp_websocket_client_handle_t esp_websocket_client_init(const esp_websocket_clie
 esp_err_t esp_websocket_client_set_uri(esp_websocket_client_handle_t client, const char *uri);
 
 /**
- * @brief      Set additional websocket headers for the client, when performing this behavior, the headers will replace the old ones
- * @pre        Must stop the WebSocket client before set headers if the client has been connected
+ * @brief      Sets additional WebSocket headers for the client
+ * @pre        The WebSocket client must be stopped before setting headers if it is already connected.
  *
- * @param[in]  client  The client
- * @param headers  additional header strings each terminated with \r\n
+ *  Notes:
+ *  - When setting headers using this method, any previous headers will be replaced.
+ *    To set multiple headers, you can concatenate them using the "\r\n" delimiter.
+ *    Alternatively, use the `esp_websocket_client_append_header` function for individual header additions.
+ *    Example: websocket_cfg.headers = "Sec-WebSocket-Key: my_key\r\nPassword: my_pass\r\n";
```

Comment:
I'd swap these two lines
```suggestion
 *    Example: websocket_cfg.headers = "Sec-WebSocket-Key: my_key\r\nPassword: my_pass\r\n";
 *    Alternatively, use the `esp_websocket_client_append_header` function for individual header additions.
```